parts/k8s/addons/kube-proxy.yaml

```diff
@@ -82,9 +82,6 @@ spec:
         operator: Exists
       containers:
       - command:
-{{- if not (IsKubernetesVersionGe "1.17.0")}}
```

Comment:
Removing this entirely breaks 1.15. If we have a 1.15 kube-proxy image, we can test that, too, otherwise maybe easiest to just change "1.17.0" to "1.16.0" in the original conditional